### ISSUE TITLE
in slr_network.py - criterion_calculation function

### ISSUE THREAD

**Issue by pallapumohankrishna** [hulianyuyy/CorrNet_Plus#12]
Why are the tensors moving to cpu here ? any reason

total_loss['ConvCTC'] = weight * self.loss['CTCLoss'](ret_dict["conv_logits"].log_softmax(-1),
                                                      label.cpu().int(), ret_dict["feat_len"].cpu().int(),
                                                      label_lgt.cpu().int()).mean()

**Comment by hulianyuyy** [comment]
It's index values, and they can be either `cpu` tensors or `cuda` tensors.
